src/arch/arm/lib/Mybuild

```diff
@@ -6,6 +6,7 @@ static module libarch extends embox.arch.libarch {
 
 	source "stack_ptr.S"
 	source "ptregs_jmp.S"
+	source "delay.S"
```

Comment:
Maybe do it as a separate module?

Reply:
OK, will put in raspberry arch lib as it is used by gpio